pkg/cmd/codespace/delete.go

```diff
@@ -54,6 +56,8 @@ func newDeleteCmd(app *App) *cobra.Command {
 	deleteCmd.Flags().StringVarP(&opts.repoFilter, "repo", "r", "", "Delete codespaces for a `repository`")
 	deleteCmd.Flags().BoolVarP(&opts.skipConfirm, "force", "f", false, "Skip confirmation for codespaces that contain unsaved changes")
 	deleteCmd.Flags().Uint16Var(&opts.keepDays, "days", 0, "Delete codespaces older than `N` days")
+	deleteCmd.Flags().StringVarP(&opts.orgName, "org", "o", "", "Select organization to delete codespace from")
+	deleteCmd.Flags().StringVarP(&opts.userName, "username", "u", "", "Username of user in organization to delete codespace from")
```

Comment:
For the `stop` command, I included a basic check that the required set of flags are included. It'd probably be worthwhile including that in this too:

https://github.com/cli/cli/pull/5812/files#diff-f668f9642bfad242337f3d0dec7fb99273ecff9ded6c08a7a3bd2f83102129a1R25-R36